packages/@uppy/thumbnail-generator/src/exif.js

```diff
@@ -0,0 +1,620 @@
+const ExifTags = {
```

Comment:
Could you please include a link to https://github.com/exif-js/exif-js with a comment that the code was adapted from exif-js and license info? Like here: https://github.com/transloadit/uppy/blob/7873f057ca1301cb642a2521361ff219fb38170b/packages/%40uppy/utils/src/prettyBytes.js#L1-L3.

Reply:
Updated!